### PR TITLE
define pre-set gateway policy

### PR DESCRIPTION
**Pull request by chris-martinelli**
defines what 5ca54fe7-c825-44b6-a464-873b685fe735 is

### CODE REVIEW

**Comment by chris-martinelli** [comment]
I'm unsure of the best wording here, but this is a policy that's automatically added so a user can't block themselves from making account changes.

File: products/cloudflare-one/src/content/faq/teams-policies-faq.md [hunk 1]
```diff
@@ -51,3 +51,6 @@ Here is a list of possible causes:
 * **Your policy is not assigned to a location.** If your policy is not assigned to a location and you send a DNS query from that location, Gateway will not apply that policy. Assign a policy to a location to make sure the desired policy is applied when you send a DNS query from that location.
 
 * **Your DoH endpoint is not a Gateway location**. Browsers can be configured to use any DoH endpoint. If you chose to configure DoH directly in your browser, make sure that the DoH  endpoint is a Gateway location. 
+
+## What is policy 5ca54fe7-c825-44b6-a464-873b685fe735?
+We disable inspection on requests to the cloudflareclient.com API in order to prevent a user from being able to put their settings into inoperable state.
```
Comment (ranbel) [inline products/cloudflare-one/src/content/faq/teams-policies-faq.md]:
```suggestion
This policy disables inspection on requests to the cloudflareclient.com API, ensuring that users cannot accidentally block themselves from making account changes. Refer to the [Global rules](/cloudflare-one/policies/filtering/http-policies/global-rules/) page for a list of all default policies.
```

**Comment by ranbel** [comment]
@chris-martinelli Is 5ca54fe7-c825-44b6-a464-873b685fe735 the same as the cloudflareclient.com policy in https://developers.cloudflare.com/cloudflare-one/policies/filtering/http-policies/global-rules/

**Review by ranbel — APPROVED** [review]

**Comment by ranbel** [comment]
Due to merge issues caused by the Pages migration, I created a new PR for these changes: #3973

**Comment by ranbel** [comment]
Due to merge issues caused by the Pages migration, I created a new PR for these changes: #3973

**Pull request closed without merging**